### ISSUE TITLE
Typescript cannot find module `packages/@markuplint/i18n/src/types`

### ISSUE THREAD

**Issue by regseb** [markuplint/markuplint#530]
# Description

Typescript cannot find module _packages/@markuplint/i18n/src/types_ from _markuplint/lib/i18n.d.ts_.

# Files

- package.json

```JSON
{
    "name": "testcase",
    "version": "1.0.0",
    "dependencies": {
        "markuplint": "2.10.1",
        "typescript": "4.8.4"
    }
}
```

- index.ts

```JavaScript
import { MLEngine } from "markuplint";

console.log(MLEngine);
```

# Commands

```Shell
npm install
npx tsc index.ts
```

# Results

```
...
node_modules/markuplint/lib/i18n.d.ts:3:25 - error TS2307: Cannot find module 'packages/@markuplint/i18n/src/types' or its corresponding type declarations.

3     listFormat?: import("packages/@markuplint/i18n/src/types").ListFormat | undefined;
                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/markuplint/lib/i18n.d.ts:4:23 - error TS2307: Cannot find module 'packages/@markuplint/i18n/src/types' or its corresponding type declarations.

4     keywords?: import("packages/@markuplint/i18n/src/types").LocalesKeywords | undefined;
                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/markuplint/lib/i18n.d.ts:5:24 - error TS2307: Cannot find module 'packages/@markuplint/i18n/src/types' or its corresponding type declarations.

5     sentences?: import("packages/@markuplint/i18n/src/types").LocalesKeywords | undefined;
                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...
```

# Additional information

I think the right import is [`@markuplint/i18n/lib/types`](https://cdn.jsdelivr.net/npm/@markuplint/i18n/lib/types.d.ts) and the problem comes from the generation of the types from the `packages/@markuplint/i18n/src/types.ts` file.

https://github.com/markuplint/markuplint/blob/59c494608543b754742eda1abd9bee1eb28e5114/packages/%40markuplint/i18n/src/types.ts#L5-L7

**Comment by YusukeHirao** [comment]
@regseb 

Thank you for your report.
I'll make sure. Wait for a soon.

**Comment by YusukeHirao** [comment]
@regseb 

It didn't reproduce in my environment.
Please tell me about your environment.
And could you make a repository?

**Comment by regseb** [comment]
My environment:

- markuplint 2.10.1
- tsc 4.8.4
- node v18.10.0
- npm 8.19.2
- Ubuntu 22.04.1 LTS

I created a repository : https://github.com/regseb/testcase-markuplint-tsc

```Shell
git clone https://github.com/regseb/testcase-markuplint-tsc.git
cd testcase-markuplint-tsc
npm install
npx tsc index.ts
```

**Comment by YusukeHirao** [comment]
@regseb 

I just released `v2.10.2` now.

You should command like the below when you use:

```sh
npm i -D @types/node
npx tsc index.ts --lib es2015,dom --target es2015 -moduleResolution node
```
